src/js/tech/flash-cache.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file flash-cache.js
+ *
+ * Auto-caching method wrapper to avoid calling through to Flash too
+ * often.
+ */
+
+/**
+ * Returns a new getter function that returns a cached value if
+ * invoked multiple times within the specified duration.
+ *
+ * @param {Function} getter the function to be cached
+ * @param {Number} cacheDuration the number of milliseconds to cache
+ * results for
+ * @return {Function} a new function that returns cached results if
+ * invoked multiple times within the cache duration
+ */
+export default function timeExpiringCache(getter, cacheDuration) {
```

Comment:
Since both uses of this function pass `100` here, I wonder if having a default (i.e. `cacheDuration = 100`) makes sense?